### ISSUE TITLE
Clear updates

### ISSUE THREAD

**Issue by konsolebox** [mihonapp/mihon#1534]
### Describe your suggested feature

Couldn't find a way to clear the Updates list; is there a way?

If no, maybe add this feature?

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.17.0](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by AntsyLich** [comment]
Why?

**Comment by konsolebox** [comment]
I want to keep it clean.

**Comment by Hepticce** [comment]
As someone who’s had their updates constantly filled with the same manga, I know how frustrating it can be to sift through repetitive entries.

**Comment by Hepticce** [comment]
Please consider this suggestion

**Comment by ireetul** [comment]
i agree there should be an option to remove entries in update section.

**Comment by Smol-Ame** [comment]
Unsave the entry from your Library & then save it again (like turning a switch off then on again).
